src/components/DateTime/DateTime.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { format, isValid } from 'date-fns';
+
+type DateTimeProps = {
+  dateString: string;
+};
+
+export const DateTime = ({ dateString }: DateTimeProps) => {
```

Comment:
V nice - though it's not (currently) returning anything with _time_ specifically so would it be more appropriately named just `Date` (or something like `FormattedDate`)?

Reply:
Fair point. I wanted it to be as generic as possible in case in future we wanted to allow props to configure a specific format

Reply:
It can't be Date since that's a native JS object

Reply:
Sorted